airflow/providers/apache/spark/hooks/spark_submit.py

```diff
@@ -546,15 +547,20 @@ def _process_spark_submit_log(self, itr: Iterator[Any]) -> None:
                 match = re.search("application[0-9_]+", line)
                 if match:
                     self._yarn_application_id = match.group(0)
-                    self.log.info("Identified spark driver id: %s", self._yarn_application_id)
+                    self.log.info("Identified spark application id: %s", self._yarn_application_id)
```

Comment:
I think this change is a breaking change because users can count on this log message, can we add it as a breaking change?

Reply:
Changing log messages is not a breaking change.

Reply:
It does not break ENOUGH. I keep on repeating that every single change breaks something that someone else **might** rely on. This is Hyrum's Law:  https://www.hyrumslaw.com/ in a full swing. 

We should just decide wheter:

a) it changes part of our Public interface: https://airflow.apache.org/docs/apache-airflow/stable/public-airflow-interface.html
b) whether it has high probability to break many of our users's workflows (one or few is not enouhg to qualify as breaking) 

In this case answer to both questions is `NO` :D